get-random-throw.js

```diff
@@ -0,0 +1,29 @@
+export function getRandomThrow() {
+    let getRandomThrow = Math.ceil(Math.random() * 3.3333);
```

Comment:
why 3.3333?

Reply:
why not 3.33333 Ryan? Why not . . . ? :)